src/test/scala/firrtlTests/ChirrtlSpec.scala

```diff
@@ -31,7 +31,7 @@ import java.io._
 import org.scalatest._
 import org.scalatest.prop._
 import firrtl.Parser
-import firrtl.ir.Circuit
+import firrtl.ir._
```

Comment:
Given the shift to a compileTest (thanks for that!), this change in import is unnecessary